src/BrowserKitDriver.php

```diff
@@ -671,6 +671,20 @@ private function getFieldPosition(\DOMElement $fieldNode)
         return 0;
     }
 
+    private function submitFormWith(Form $form, \DOMElement $submit_element)
+    {
+
+        if ($formaction = $submit_element->getAttribute('formaction')) {
```

Comment:
Not sure about others, but assignment in condition is bad idea, because it looks like somebody forgot to add `==` instead of `=`.

Probably better not to use such approach (assign above IF and check inside IF).